src/D2L.CodeStyle.Analyzers/Immutability/ImmutableDefinitionChecker.cs

```diff
@@ -70,7 +70,14 @@ ImmutabilityContext context
 		/// values.
 		/// </remarks>
 		public bool CheckMember( ISymbol member ) {
-			if ( MutabilityAuditor.IsAudited( member, out var location ) ) {
+			if ( MutabilityAuditor.IsAudited( member, out var location, out var diagnostic ) ) {
+
+				// If they have one of the auditing attributes,
+				// error if a diagnostic occurred

```

Comment:
super nit: we wrap at around 80 chars :)

So if we have a conflict we will report it but carry on as if there was no conflict (i.e. assume this is audited, but also check that an audit is even warranted in the first place)... that makes sense to me.